.github/actions/create-featurenet/action.yml

```diff
@@ -148,21 +148,24 @@ runs:
           pr_image_tag="fe-${{ steps.get-ref-properties.outputs.branch-name-for-argo-with-sha }}"
           fnet_aleph_node_image="${{ inputs.ecr-public-registry }}feature-env-aleph-node:${pr_image_tag}"
           fnet_bootstrap_chain_node_image="${{ steps.get-ref-properties.outputs.sha }}"
+          fnet_create_hook="false"
         elif [[ "${{ inputs.featurenet-aleph-node-image }}" == "testnet" || \
             "${{ inputs.featurenet-aleph-node-image }}" == "mainnet" ]]; then
           ecr_image_tag=$(echo "${{ steps.get-node-commit-sha.outputs.sha }}")
           fnet_aleph_node_image="${{ inputs.ecr-public-registry }}aleph-node:${ecr_image_tag}"
           fnet_bootstrap_chain_node_image="${{ inputs.featurenet-aleph-node-image }}"
+          fnet_create_hook="false"
         else
           fnet_aleph_node_image="${{ inputs.ecr-public-registry }}aleph-node:${{ inputs.featurenet-aleph-node-image }}"
           fnet_bootstrap_chain_node_image="none"
+          fnet_create_hook="true"
```

Comment:
So that is `true` only for Updating existing Updatenet, right?  Why is it `false` when featurenet is created of a PR branch (first `if` above where `"${{ inputs.featurenet-aleph-node-image }}" == ""`)?